kafka/src/main/java/io/micronaut/configuration/kafka/ConsumerRegistry.java

```diff
@@ -95,4 +117,18 @@
      * @param id The id of the consumer
      */
     void resume(@NonNull String id);
+
+    /**
+     * Resume the consumer for the given ID to consume from the given topic partitions.
+     * Note that this method will request that the consumer is resumed, however
+     * does not block until the consumer is actually resumed.
+     * You can use the {@link #isPaused(String, Collection<TopicPartition>)} method to
+     * establish when the consumer has actually been resumed to consume from the given topic partitions.
+     *
+     * @param id The id of the consumer
+     * @param topicPartitions The topic partitions to pause consuming from
+     */
+    void resume(@NonNull String id, Collection<TopicPartition> topicPartitions);
```

Comment:
Can you add `@NonNull` or `@Nullable` to the second arg and add `@since` to the javadoc